nym-vpn-core/crates/nym-vpn-lib/src/platform/mod.rs

```diff
@@ -261,6 +261,12 @@ pub fn resetDeviceIdentity(path: String) -> Result<(), VpnError> {
     RUNTIME.block_on(account::reset_device_identity(&path))
 }
 
+#[allow(non_snake_case)]
+#[uniffi::export]
+pub fn getDeviceIdentity(path: String) -> Result<String, VpnError> {
+    RUNTIME.block_on(account::get_account_id(&path))
```

Comment:
This will give you the account id, the one that is associated with your account in the webapp. Unfortunately I think you'll need to create a `account::get_device_id` function in `account.rs`

Reply:
In `vpn_service.rs` there is a `handle_get_device_identity()` that should provide some info how to get the device identity. It's very similar to how the account identity is retrieved

Reply:
<img width="617" alt="Screenshot 2024-12-05 at 16 17 13" src="https://github.com/user-attachments/assets/1e9bd349-9f24-4563-a265-a5a841492bb7">

There is one already?

Reply:
That gives you the account_id, but next to that function there is a get_device_id one too. You can't call that one from uniffi, but it should give some pointers to how you can add a get_device_id in uniffi by starting from get_account_it

Reply:
Let me try doing that 👍